### ISSUE TITLE
Unset field warnings on Upload New Media page

### ISSUE THREAD

**Issue by noah-schechter** [milesmcc/atlos#26]
If either or both fields on the first Upload New Media page are unset and I press the "Upload media" button, the 'can't be blank' warning for the applicable fields briefly pops up then disappears. I think they should stay visible until the fields are set.

**Comment by noah-schechter** [comment]
I tried to replicate this for a screen recording and the blank warnings were disappearing every time I pressed the "Upload media" button with unset fields. I'll try to figure out a better description of this pattern.

**Comment by milesmcc** [comment]
Yeah I think I've seen this. Not totally sure why it's happening. Can look for a fix eventually...
